CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Explainability in hybrid query ([#970](https://github.com/opensearch-project/neural-search/pull/970))
 - Support new knn query parameter expand_nested ([#1013](https://github.com/opensearch-project/neural-search/pull/1013))
 - Implement pruning for neural sparse ingestion pipeline and two phase search processor ([#988](https://github.com/opensearch-project/neural-search/pull/988))
+- Allow empty string for field in field map ([#1041](https://github.com/opensearch-project/neural-search/pull/1041))
```

Comment:
```suggestion
- Allow empty string for field in field map of text embedding processor ([#1041](https://github.com/opensearch-project/neural-search/pull/1041))
```

Reply:
It can also be "Support empty string for fields in text embedding processor"

Reply:
As discussed offline, will use `Support empty string for fields in text embedding processor` as new CHANGELOG message